framework/wazuh/core/exception.py

```diff
@@ -127,6 +127,10 @@ class WazuhException(Exception):
                'remediation': f'To solve this issue, please enable agents higher versions in the API settings: '
                               f'https://documentation.wazuh.com/{DOCU_VERSION}/user-manual/api/'
                               f'configuration.html#agents'},
+        1130: {'message': 'Indexer section modification attempt detected',
+               'remediation': f'To solve this issue, please enable indexer configuration changes in the API settings: '
+                              f'https://documentation.wazuh.com/{DOCU_VERSION}/user-manual/api/'
+                              f'configuration.html#indexer'},
```

Comment:
We should reuse the exception `1127` instead, adding an extra message to let the user know which is the protected section that was modified, as in:
 https://github.com/wazuh/wazuh/blob/8e3eb05d8e64fdf20d2be4c71891dc9ba8569ec6/framework/wazuh/core/utils.py#L941